src/test/Oppfolging.test.tsx

```diff
@@ -0,0 +1,199 @@
+import {Bruker, PeriodeData} from "../utils/typer";
+import {HenvendelseList} from "../view/HenvendelseList";
+import {DialogInputBox} from "../view/DialogInputBox";
+import {DialogHeader} from "../view/DialogHeader";
+import {AlertStripeContainer} from "../view/AlertStripeContainer";
+import {AlertStripeAdvarsel} from "nav-frontend-alertstriper";
+import {DialogOverview} from "../view/DialogOverview";
+import {DialogOverviewHeader} from "../view/DialogOverviewHeader";
+import {DialogPreview} from "../view/DialogPreview";
+import {DialogHeaderCheckboxes} from "../view/DialogHeaderCheckboxes";
+
+Enzyme.configure({ adapter: new Adapter() });
+
+const userInfo: Bruker = {id:"010101", erVeileder: true,erBruker:false};
+const oppfPerioder: PeriodeData[] = [];
+const oppfolgingData = {
+    fnr: null,
+    veilederId: "101010",
+    reservasjonKRR: false,
+    manuell: false,
+    underOppfolging: true, //eller false
+    underKvp: false,
+    oppfolgingUtgang: null,
+    gjeldendeEskaleringsvarsel: null,
+    kanStarteOppfolging: false,
+    avslutningStatus: null,
+    oppfolgingsPerioder: oppfPerioder,
+    harSkriveTilgang: true,
+    kanReaktiveres: false,
+    inaktiveringsdato: '2018-08-31T10:46:10.971+01:00',
+};
+
+const dialoger = [
+    {
+        id: '1',
+        aktivitetId: '1',
+        overskrift: 'Memes',
+        sisteTekst: 'Hei. Hva er status her? Har du finnet Kaptain Sabeltann?',
+        sisteDato: '2018-01-28T12:48:56.097+01:00',
+        opprettetDato: '2018-02-27T12:48:56.081+01:00',
+        historisk: false,
+        lest: true,
+        venterPaSvar: false,
+        ferdigBehandlet: false,
+        lestAvBrukerTidspunkt: null,
+        erLestAvBruker: false,
+        henvendelser: [
+            {
+                id: '1',
+                dialogId: '1',
+                avsender: 'VEILEDER',
+                avsenderId: 'Z123456',
+                sendt: '2018-02-27T12:48:56.097+01:00',
+                lest: true,
+                tekst:
+                    'Hei. Hva er status her? Har du finnet Kaptain Sabeltann?',
+            },
+            {
+                id: '2',
+                dialogId: '1',
+                avsender: 'BRUKER',
+                avsenderId: '0102030405',
+                sendt: '2018-02-28T12:48:56.097+01:00',
+                lest: true,
+                tekst: 'Hei. Leter enda på sjøen :)',
+            },
+        ],
+        egenskaper: [],
+    }];
+
+describe('<AlertStripeContainer/>', () => {
+
+    test("Bruker uten oppf.perioder og ikke under oppf. viser en advarsel - veileder.", () => {
+        userInfo.erVeileder=true;
+        oppfolgingData.underOppfolging = false;
+        oppfolgingData.oppfolgingsPerioder = [];
+        jest.spyOn(AppContext, 'useUserInfoContext').mockImplementation(() => userInfo);
+        jest.spyOn(AppContext, 'useOppfolgingContext').mockImplementation(() => oppfolgingData);
+        const wrapper = shallow(<AlertStripeContainer/>);
```

Comment:
Testet dere å bruke `mount(<Provider><AlertStripeContainer /></Provider>)`
Vet ikke hvordan det er med `useContext`, men om jeg ikke husker helt feil så fungerte det med det gamle context-apiet. 

Tenker det uansett er en grei måte å løse det på. Men da hadde vi sluppet å bruke `mockImplementation`

Reply:
@nutgaard Ja, det fungerte før men ikke med kombinasjonen useContext  og shallow render. Se for eksempel: https://github.com/airbnb/enzyme/issues/2176

Reply:
Med `shallow` forventet jeg nesten at det ikke skulle fungere, men `mount` setter opp ting nesten som `React.render` så derfor jeg tenkte det kunne fungert